tools/acorn-optimizer.mjs

```diff
@@ -857,9 +871,19 @@ function emitDCEGraph(ast) {
           }
         }
       }
+    } else if (node.type === 'Property' && node.method) {
```

Comment:
`assert(specialScopes > 0)` ?

Reply:
(Sorry just saw the one below.. probably don't need both)

Reply:
Why, I think both make equal sense?

Reply:
lgtm either way.